CHANGES.rst

```diff
@@ -20,6 +20,9 @@ Version 2.0.1
 
 Released 2021-05-21
 
+-   Renamed a variable self._bg_loading_exc_info to self._bg_loading_exc
+    and used that variable to store Exception instead of storing
+    Exception info and also wrote Exception testcase for it. :pr:`4169`
```

Comment:
Don't add changelog entries for an already-released version... add them for 2.0.2 (at the end of that list)

Reply:
Done